packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -293,6 +295,14 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         setLogo(logo);
     }, [logo]);
 
+    useEffect(() => {
+        if (currentLocation) {
```

Comment:
If a customer is using another JS SDK version (we allow that - we only load the JS SDK if the mapsindoors namespace doesn't exist), this will probably throw an error. Should we handle this better?

Reply:
Added a check if the method exists in the SDK. ✅